kura/org.eclipse.kura.net.admin/src/main/java/org/eclipse/kura/net/admin/visitor/linux/HostapdConfigWriter.java

```diff
@@ -118,10 +118,11 @@ private void writeConfig(NetInterfaceConfig<? extends NetInterfaceAddressConfig>
                             }
                         }
 
-                        if (netInterfaceStatus == NetInterfaceStatus.netIPv4StatusDisabled) {
+                        if (netInterfaceStatus == NetInterfaceStatus.netIPv4StatusDisabled
```

Comment:
maybe this check, that is the same in other files, can be externalised into a method like isDisabled or isManaged that could simplify the reading

Reply:
done